src/main/java/com/work/management/service/log/impl/LoggingServiceImpl.java

```diff
@@ -0,0 +1,77 @@
+package com.work.management.service.log.impl;
+
+import com.google.common.collect.ImmutableMap;
+import com.work.management.service.log.LoggingService;
+import java.util.Enumeration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Stream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+@Service
+final class LoggingServiceImpl implements LoggingService {
+
```

Comment:
Add an empty package private constructor, to limit construction of this class to only this package.

Reply:
You can even the empty constructor with @AutoWired annotation, to showcase future developers to try not to call this constructor instead use Spring AutoWiring to have the istance.

So mark the constructor as AutoWired also. (Although its not doing much here but at it here).